plugins/symbol-processor/src/main/kotlin/org/jetbrains/dataframe/ksp/GetQualifiedNameOrThrow.kt

```diff
@@ -1,7 +1,20 @@
 package org.jetbrains.dataframe.ksp
 
 import com.google.devtools.ksp.symbol.KSDeclaration
+import org.jetbrains.kotlinx.dataframe.impl.codeGen.quoteIfNeeded
 
-fun KSDeclaration.getQualifiedNameOrThrow(): String {
-    return (qualifiedName ?: error("@DataSchema declaration ${simpleName.asString()} at $location must have qualified name")).asString()
-}
+fun KSDeclaration.getQualifiedNameOrThrow(): String =
```

Comment:
isn't this commit merged already? is it intentional or something strange happened?

Reply:
huh you're right. That's odd